### ISSUE TITLE
Images in blob storage

### ISSUE THREAD

**Issue by tjcrone** [tjcrone/rte-camhd#7]
@al3623 and @gregyhj, I am considering different naming conventions for image data in blob storage. We have a few options. One of them is to use the mov timestamp along with the frame number, so something like this using epoch:

1469750401-00003463

or something using the timestamp of the image including microseconds:

20160729000156626043

or something of a hybrid:

20160729000156-00003463

or to use some other variant of the above including underscores or something. Do either of you have any thoughts on this? I am leaning toward the last option primarily because it has the most meaning when just looking at the name, and is the most precise because it contains our most precise possible stamp since all we have is the epoch seconds and the frame number which we use to calculate the actual timestamp of the frame using frame rate. What do you think?

**Comment by gregyhj** [comment]
Yes, the last option seems fine: some kind of punctuation marker in the middle is necessary since down the road people will probably want to be able to eyeball the names and get some basic info at a glance.  

Thinking about it, mov-frame (first option) seems marginally better: at a glance, we can efficiently tell which frames are from which videos, and hence which videos have been "processed" for blob storage and which haven't. The last convention doesn't give any extra information that the first option can't (since we would just need to do the calculation to get the exact microsecond), and actually loses information as compared to the first option (since now we aren't able to identify which mov file it came from).

**Comment by al3623** [comment]
I agree with Greg. I think the first is a little more intuitive and pithy and should be able to convey unique information.
